src/main/java/org/folio/circulation/domain/policy/FixedDueDateSchedules.java

```diff
@@ -17,18 +17,30 @@
 
 public class FixedDueDateSchedules {
   private final List<JsonObject> schedules;
+  private final DateTimeZone timeZone;
 
-  FixedDueDateSchedules(List<JsonObject> schedules) {
+  FixedDueDateSchedules(List<JsonObject> schedules, DateTimeZone timeZone) {
     this.schedules = schedules;
+    this.timeZone = timeZone;
   }
 
-  static FixedDueDateSchedules from(JsonObject representation) {
+  static FixedDueDateSchedules from(JsonObject representation, DateTimeZone timeZone) {
     //TODO: Replace this with better check
     if (representation == null) {
       return new NoFixedDueDateSchedules();
     } else {
       return new FixedDueDateSchedules(JsonArrayHelper.toList(
-        representation.getJsonArray("schedules")));
+        representation.getJsonArray("schedules")), timeZone);
+    }
+  }
+
+  // for test only
+  static FixedDueDateSchedules from(JsonObject representation) {
+    if (representation == null) {
+      return new NoFixedDueDateSchedules();
+    } else {
+      return new FixedDueDateSchedules(JsonArrayHelper.toList(
+        representation.getJsonArray("schedules")), DateTimeZone.UTC);
```

Comment:
Similarly, why is UTC hardcoded here?

Reply:
As far as I understood from the code, this method is called in "LoanPolicy" only for tests.
And at the moment it makes sense to hardcode.

Reply:
Why not just use the `static FixedDueDateSchedules from(JsonObject representation, DateTimeZone timeZone)` method and pass UTC in from the tests?